src/bors/command/parser.rs

```diff
@@ -127,10 +134,29 @@ fn parse_list<'a>(
     Some(Ok(result))
 }
 
+pub fn parse_key_value(input: &str) -> Result<HashMap<String, String>, String> {
+    let mut kv_pairs = HashMap::new();
+
+    for pair in input.split_whitespace() {
+        if let Some((key, value)) = pair.split_once('=') {
+            kv_pairs.insert(key.to_string(), value.to_string());
+        }
+    }
+
+    if kv_pairs.is_empty() {
+        return Ok(HashMap::new());
+    }
```

Comment:
```suggestion
```
This is not needed, you can just return `Ok(kv_pairs)` directly.

Reply:
Perfect !